event-based-simulators/main/event_based_simulators.py

```diff
@@ -358,6 +371,7 @@ def load(filename):
 if CREATE_PROFILES.lower() == "true":
     [oeeAPI.create_and_activate_profile(id, ProfileCreateMode.CREATE_IF_NOT_EXISTS) 
         for id in oeeAPI.get_simulator_external_ids()]
+    os.system("python profile_generator.py -cat")
```

Comment:
@mvoigt-sag I am not sure about that. One of the profiles requires the presence of the categories. So creating one without the other does not make much sense.

Reply:
Currently we are using ids 1 and 2. Looks like current implementation takes care of POST vs PUT the categories, but if there are categories already the might be overwritten, correct?

Reply:
Yes, its checked if the managed object of that type exists and in case it exists it will be overwritten. If there are more than one object of that type, it will log an error.

Reply:
OK, so we should maybe use other IDs for the categories. maybe some higher numbers like 10001 & 10002.